central/private/server.go

```diff
@@ -0,0 +1,49 @@
+package private
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/NYTimes/gziphandler"
+	"github.com/stackrox/rox/pkg/env"
+)
+
+// HTTPServer is a HTTP server to serve private functionality.
+type HTTPServer struct {
```

Comment:
Do you mean you want to serve `/metrics` on the `9095` port, or do you want to include them in `diagnostics` response?

Reply:
Essentially talking about this part here which is done within the default gRPC server: https://github.com/stackrox/stackrox/blob/cd8bfd0725d47ee391272d76db32272eb637c99d/pkg/grpc/server.go#L298-L300 which will AFAICT ensure that we record metrics for those API paths as well (like status code failures etc.) in case we want to create alerts for those. As said it's probably optional but something to think about in case its realtively easy to do.

Reply:
Ah, I agree, let's do that